hlx_statics/blocks/iframe/iframe.js

```diff
@@ -90,7 +90,8 @@ export default async function decorate(block) {
     const penpalScript = addExtraScriptWithReturn(document.body, 'https://unpkg.com/penpal@^6/dist/penpal.min.js');
     const iframeSrc = block.querySelector('a');
     const iframeContainer = block.parentElement;
-    const iframe = createTag('iframe', { class: 'iframe-container', 'src': iframeSrc.href, 'id': 'penpalIframe' });
+    const title = block?.parentElement?.parentElement?.attributes.getNamedItem('data-title')?.value;
```

Comment:
Need a `?` at the end of `block?.parentElement?.parentElement?.attributes`

Reply:
oop, my fault, I fixed it locally but wanted to await pushing to see if we needed to create a new block as well.